### PR TITLE
📝  Source Stripe: documentation update (MLP requirements)

### PR DESCRIPTION
**Pull request by bazarnov**
## What
Due to MLP Requirements and UI/UX improvement we need to make small corrections, based on [this document](https://docs.google.com/spreadsheets/d/1wCPX3eBBPFRI5sMeK4LBjooDs3bFpAa9Fr2os0S0P3g/edit#gid=1288588270).

## 🚨 User Impact 🚨
No impact expected.

## Pre-merge Checklist
Expand the relevant checklist and delete the others.

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- [X] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [X] Documentation updated
    - [X] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [X] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)
- [ ] After the new connector version is published, connector version bumped in the seed directory as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
- [ ] Seed specs have been re-generated by building the platform and committing the changes to the seed spec files, as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)

</details>

### CODE REVIEW

**Review by misteryeo — APPROVED** [review]
Just had one comment to update otherwise LGTM!

File: airbyte-integrations/connectors/source-stripe/source_stripe/spec.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
         "title": "Lookback Window (in days)",
         "default": 0,
         "minimum": 0,
-        "description": "When set, the connector will always reload data from the past N days, where N is the value set here. This is useful if your data is updated after creation.",
+        "description": "When set, the connector will always reload data from the past N days, where N is the value set here. This is useful if your data is updated after creation. More info: <a href=\"https://docs.airbyte.com/integrations/sources/stripe#requirements\">here</a>",
```
[the remark this answers is not in the document]
Comment (bazarnov) [reply airbyte-integrations/connectors/source-stripe/source_stripe/spec.json:36]: Got it, corrected.

**Pull request merged**